packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1050,7 +1051,7 @@ class EditableText extends StatefulWidget {
   /// {@template flutter.widgets.editableText.autocorrect}
   /// Whether to enable autocorrection.
   ///
-  /// Defaults to true.
+  /// Defaults to true, false on iOS if [autofillHints] contains password related hints.
```

Comment:
What do you think of this? I don't feel strongly, feel free to keep as-is.

```suggestion
  /// False on iOS if [autofillHints] contains password-related hints, otherwise true.
```

Reply:
Thank you!